Benchmark-Models/Laske_PLOSComputBiol2019/measurementData_Laske_PLOSComputBiol2019.tsv

```diff
@@ -0,0 +1,43 @@
+observableId	simulationConditionId	measurement	time	noiseParameters
+RM5	virus_infection	16.4862319433747	1	sigma_RM5
+RM5	virus_infection	137.529004093062	2	sigma_RM5
+RM5	virus_infection	5927.73630370154	3	sigma_RM5
+RM5	virus_infection	26070.7777033051	4	sigma_RM5
+RM5	virus_infection	36165.1722001919	5	sigma_RM5
+RM5	virus_infection	39708.6354393543	6	sigma_RM5
+RM5	virus_infection	33557.6057996387	7	sigma_RM5
+RM5	virus_infection	26851.6819153912	8	sigma_RM5
+RM5	virus_infection	24129.7114856115	9	sigma_RM5
+RM5	virus_infection	15503.6668243303	10	sigma_RM5
+RVSegTot	virus_infection	535.171384487193	1	sigma_RV
+RVSegTot	virus_infection	505.340542126865	2	sigma_RV
+RVSegTot	virus_infection	2257.56566823967	3	sigma_RV
+RVSegTot	virus_infection	13599.6998388987	4	sigma_RV
+RVSegTot	virus_infection	34431.2425312543	5	sigma_RV
+RVSegTot	virus_infection	52925.7096508491	6	sigma_RV
+RVSegTot	virus_infection	64547.3268688838	7	sigma_RV
+RVSegTot	virus_infection	74091.3081045583	8	sigma_RV
+RVSegTot	virus_infection	88032.0436517163	9	sigma_RV
+RVSegTot	virus_infection	97993.174580701	10	sigma_RV
+RCSegTot	virus_infection	0.316575722115733	1	sigma_RC
```

Comment:
if i saw that correctly from your presentation, there are molecule number per cell as unit? how can we have then decimal numbers? and specially 0.31 molecules per cell?

Reply:
The measurements in the table are averages from 4 measurements in total. To measure these RNAs they performed RT-qPCR from which an exact number of molecules cannot be directly measured. To calculate molecules per cell they first had to create a calibration curve where they could then estimate the approximate number of molecules per cell (the authors refer to this article for calculations of molecule per cell: Frensing, T., Kupke, S.Y., Bachmann, M. et al. Influenza virus intracellular replication dynamics, release kinetics, and particle morphology during propagation in MDCK cells. Appl Microbiol Biotechnol 100, 7181–7192 (2016). https://doi.org/10.1007/s00253-016-7542-4 for more details).

Reply:
oki thanks for clarifying 👍🏼